packages/remix-dev/__tests__/cli-test.ts

```diff
@@ -230,7 +230,7 @@ describe("remix CLI", () => {
       await interactWithShell(proc, [
         { question: /Where.*create.*app/i, type: [projectDir, ENTER] },
         { question: /What type of app/i, answer: /basics/i },
-        { question: /Where.*deploy/i, answer: /express/i },
+        { question: /Where.*deploy/i, answer: /remix/i },
```

Comment:
Ignore this unit test fix - this was fixed in `dev` and cherry-picked into `main` and `release-next`.  This branch has the `main` SHA since I merged `main` in - but this code is already in `release-next` so it's not actually changing anything.